src/components/SigninForm/SigninForm.jsx

```diff
@@ -0,0 +1,198 @@
+import React from 'react'
+import {
+    Box,
+    Typography,
+    Button,
+    TextField,
+    InputAdornment,
+    Checkbox,
+    Link,
+    styled,
+} from '@mui/material'
+import backgroundImage from 'Images/heroSen.jpg'
+import PersonIcon from '@mui/icons-material/Person'
+import LockRoundedIcon from '@mui/icons-material/LockRounded'
+import RadioButtonUncheckedIcon from '@mui/icons-material/RadioButtonUnchecked'
+import CheckCircleOutlineIcon from '@mui/icons-material/CheckCircleOutline'
+
+const InputText = styled(TextField)(() => ({
+    outline: '0.1rem solid #69AD28',
+    borderRadius: '0.5rem',
+}))
+
+const iconStyle = {
+    color: '#69AD28',
+    fontSize: '2rem',
+}
+
+export default function SigninForm() {
+    const inputItems = [
+        { icon: <PersonIcon sx={iconStyle} />, text: 'Email' },
+        { icon: <LockRoundedIcon sx={iconStyle} />, text: 'Mật khẩu' },
+    ]
+    return (
+        <Box>
+            <Box
+                sx={{
+                    position: 'relative',
+                    backgroundImage: `url(${backgroundImage})`,
+                    backgroundPosition: 'center',
+                    backgroundSize: 'cover',
+                    backgroundRepeat: 'no-repeat',
+                    width: '100vw',
+                    height: '100vh',
+                    filter: `blur(5px)`,
+                    zIndex: -1,
```

Comment:
There is alot of sx style here, like i say in first comment, you need to add another styled file and then import to use.